installation/raspberry-pi-desktop/boot-options/README.md

```diff
@@ -0,0 +1,39 @@
+# Raspberry Pi Desktop boot options
+
+When booting the Raspberry Pi Desktop image, you will be presented with a
+selection screen offering a number of options for how to run
+or install it:
+
+![Raspberry Pi Desktop boot options](images/rpd-boot-options.png)
+
+## Options
+
+1. **Run with persistence**
+    - Live boot the OS with persistence of files and programs — any changes made
+    will remain after reboot
+1. **Run and reset persistence**
+    - Reset any changes made, and live boot the OS with persistence of files
+    thereafter
+1. **Run without persistence**
+    - Run the OS with no persistence of changes
+1. **Install**
+    - Install the OS fully using a command-line installer
+1. **Graphical install**
+    - Install the OS fully using a graphical installer
+1. **Advanced options**
```

Comment:
Are there plans to replace the `???`?

Reply:
Hmmm, hadn't realised there were sections missing.  @bennuttall Should we remove until have the required text, or fill in now?